deploy/complete/helm-chart/mushop/charts/edge/values.yaml

```diff
@@ -2,7 +2,7 @@ replicaCount: 1
 
 image:
   repository: iad.ocir.io/oracle/ateam/mushop-edge-router
-  tag: 1
+  tag: 1.1.0
```

Comment:
Should not be 1.1?

Reply:
Same thought as above, mostly to be consistent